app/Http/Controllers/MessagesController.php

```diff
@@ -92,7 +92,9 @@ public function sendMessage(Message $message)
     {
         // Get competition with activity.
         $competitionId = request('competition_id');
-        // $competition = null;
+
+        $contest = Contest::find(request('contest_id'));
+        $this->manager->appendCampaign($contest);
```

Comment:
yea that is probably safer...but it actually works /shrug

I was following what was done here: 

https://github.com/DoSomething/gladiator/blob/f4fda9c508118e9051a60c532bf5d7041d7487a1/app/Http/Controllers/Api/UsersController.php#L96

I will update though.